### PR TITLE
Ft/add Todos Controllers and Routes

### PR DESCRIPTION
**Pull request by fojuri**
- Add Todos controllers and routes
- Add updateUser controller

### CODE REVIEW

File: controllers/todo.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        });
+    });
+  },
+
+  deleteUserTodo(req, res) {
+    Todo.findById(req.params.id).then((todo) => {
+      if (todo.userId === req.decoded.id) {
+        todo.destroy().then(() => res.send({ success: true, message: 'Todo deleted Successfully' }));
+      } else {
+        res.status(400).send({ success: false, message: 'You do not have permission to delete this Document' });
```
Comment (oreoluwade) [inline controllers/todo.js]: `You do not have permission to delete this Todo`

File: controllers/user.js [hunk 2]
```diff
@@ -37,6 +37,33 @@ const Users = {
       }
     }).catch(() => res.status(500).send({ success: false, message: 'Unexpected error occured' }));
   },
+
+  updateUser(req, res) {
```
Comment (oreoluwade) [inline controllers/user.js:42]: This does not account for updating an email? 🤔

Comment (fojuri) [reply controllers/user.js:42]: I wanted to prevent that from happening but I think it makes sense to include it

**Comment by fojuri** [comment]
@oaboluwarin , PR  has been updated. Thanks.

**Pull request merged**